### PR TITLE
Fix Issue #805: Added check in summarize_corpus

### PR DESCRIPTION
**Pull request by anmol01gulati**
* Added check in sumarize_corpus to reject summarizing if number of nodes in graph is less than 3
* This resolves Issue #805

### CODE REVIEW

**Comment by tmylk** [comment]
Thanks for taking this up. Please add a test for the warning.

File: gensim/summarization/summarizer.py [hunk 1]
```diff
@@ -158,6 +158,11 @@ def summarize_corpus(corpus, ratio=0.2):
     _set_graph_edge_weights(graph)
     _remove_unreachable_nodes(graph)
 
+    # If the number of nodes < 3, the function ends.
```
Comment (tmylk) [inline gensim/summarization/summarizer.py]: Please add a better comment explaining original error

**Comment by tmylk** [comment]
Looks good. Please add a note in the CHANGELOG for 0.13.3 release and will merge.

**Comment by anmol01gulati** [comment]
I've made the changes now.
The _test_get_texts_returns_generator_of_lists (gensim.test.test_wikicorpus.TestWikiCorpus)_ test hangs in python 2.6 on Travis. I'm not really able to find out about this. 
@tmylk Any suggestions?

**Comment by tmylk** [comment]
Hi @anmol01gulati , it is some travis weirdness with `gensim.test.test_wikicorpus.TestWikiCorpus`. Just add/remove blank line to the file and it should run fine as in https://github.com/RaRe-Technologies/gensim/pull/887

**Comment by tmylk** [comment]
Great. Thanks for the pr!

File: gensim/summarization/summarizer.py [hunk 2]
```diff
@@ -158,6 +158,11 @@ def summarize_corpus(corpus, ratio=0.2):
     _set_graph_edge_weights(graph)
     _remove_unreachable_nodes(graph)
 
+    # Cannot calculate eigenvectors if number of unique words in text < 3. Warns user to add more text. The function ends.
+    if len(graph.nodes()) < 3:
+        logger.warning("Please add more sentences to the text. The number of reachable nodes is below 3")
+        return
```
Comment (piskvorky) [inline gensim/summarization/summarizer.py]: This doesn't match the function signature (`Returns a list`). Either raise an exception, or return an empty list -- this is just nasty unexpected gotcha for users.

**Pull request merged**